docs/pages/upgrade-from-legacy-V2.md

```diff
@@ -9,7 +9,7 @@ This document is for you if you are upgrading from a legacy XMTP V2 SDK. Legacy
 
 :::tip[Upgrading from a legacy XMTP V3 SDK?]
```

Comment:
Primarily just updates to the version numbers - version numbers are placeholders until we have the actual versions.